docs/static/upgrading.asciidoc

```diff
@@ -66,12 +66,23 @@ Before upgrading Logstash, remember to read the <<breaking-changes,breaking chan
 If you are installing Logstash with other components in the Elastic Stack, also see the
 {stack-ref}/index.html[Elastic Stack installation and upgrade documentation].
 
-If you are using the default mapping templates in Logstash, to continue using these after migrating Elasticsearch to 6.0, you must override the existing template with the 6.x template.
- This can be done by starting a pipeline with the `overwrite_template => true` option in the Elasticsearch output definition in the Logstash config.
+==== Issue with Logstash 6.0.0 `document_type` when writing to Elasticsearch 6.x
 
-Note that multiple doctypes are no longer supported in Elasticsearch 6.0. Please refer to
- {ref}/removal-of-types.html[Removal of mapping types] and {ref}/breaking-changes.html[Breaking changes] for more information.
+We’d like to alert users to behavior in Logstash 6.0.0 that can cause errors writing to Elasticsearch 6.0+ clusters. When the `type` field has different values for different events per Elasticsearch index Logstash will encounter errors while indexing documents. These errors look something like the example below, which has been shortened from the full message. This bug is likely to be encountered by users using either Beats, or multiple inputs with different `type` values.
 
+`[2017-11-21T14:26:01,991][WARN ][logstash.outputs.elasticsearch] Could not index event to Elasticsearch. {:status=>400,  :response=>{"error"=>{"reason"=>"Rejecting mapping update to [myindex] as the final mapping would have more than 1 type: [type1, type2]"}}}}`
+
+The workaround for Logstash 6.0.0 is to add the setting `document_type => doc` to the Elasticsearch output configuration. Logstash 6.0.1+ will not require this workaround as that will become the new default behavior for Logstash when communicating with Elasticsearch 6.0+.
```

Comment:
> Logstash 6.0.1+ will not require this workaround as that will become the new default behavior for Logstash when communicating with Elasticsearch 6.0+.

I'd be a little concerned to promise this, is there a problem with adding this sentence in present tense when we implement and deliver the 6.0.1 release?

Reply:
I would call it a blocker for LS 6.0.1, but you're right, it probably is prudent not to promise this until its released. I'll replace it with 'a future version'